actors/common/src/test/java/org/sunbird/learner/actors/otp/OTPActorTest.java

```diff
@@ -144,27 +149,51 @@ private void verifyOtpFailureTest(boolean isPhone, Response mockedCassandraRespo
         .thenReturn(mockedCassandraResponse);
     subject.tell(request, probe.getRef());
     ProjectCommonException exception =
-        probe.expectMsgClass(duration("10 second"), ProjectCommonException.class);
+        probe.expectMsgClass(duration("100 second"), ProjectCommonException.class);
     Assert.assertTrue(
         ((ProjectCommonException) exception)
             .getCode()
             .equals(ResponseCode.errorInvalidOTP.getErrorCode()));
   }
 
+  private void verifyOtpFailureTest(boolean isPhone, Response mockedCassandraResponse) {
```

Comment:
have a look into this method , it either it should be @Test or removed Assert.

Reply:
this method is getting called from test method